scripts/google/covid_mobility/covidmobility.txt

```diff
@@ -0,0 +1,10 @@
+import: {
+  import_name: "covid19mobility"
+  provenance_url: "https://www.google.com/covid19/mobility/"
+  provenance_description: "Shows how visits and length of stay at different places have changed over time compared to a baseline"
+  mcf_url: "gs://unresolved_mcf/covid_mobility/unresolved_mcf.mcf"
```

Comment:
What are these different files? Why are we adding unresolved MCFs to manifest?

Reply:
Sorry, fixed.